src/renderer/git/log.js

```diff
@@ -1,31 +1,23 @@
 import nodegit from "nodegit";
 
-const log = async path => {
+const log = async (path, count = 50) => {
 	const repo = await nodegit.Repository.open(path);
-	const commit = await repo.getHeadCommit();
 
-	// History returns an event.
-	const history = commit.history(nodegit.Revwalk.SORT.TIME);
+	const newRevWalk = repo.createRevWalk();
 
-	let logs = [];
+	const commit = await repo.getHeadCommit();
 
-	// History emits "commit" event for each commit in the branch's history
-	history.on("commit", function(commit) {
-		let objectCommit = {
-			author_email: commit.author().email(),
-			author_name: commit.author().name(),
-			body: commit.body(),
-			date: commit.date(),
-			hash: commit.sha(),
-			message: commit.message()
-		};
+	const result = newRevWalk.push(commit.sha());
 
-		logs.push(objectCommit);
-	});
+	if (result !== 0) {
+		console.log("Error: " + result);
```

Comment:
I don't know what to do if are any error. What is your suggestion?

![Documentation of NodeGit about error](https://user-images.githubusercontent.com/52673485/66869407-b9448b80-ef75-11e9-8553-9df6b899cdcd.png)
)

Reply:
Well, we do need to build a separate notification section which includes to trigger notifications, alerts and more from anywhere.

But for now we can just log the error messages to console.